utils/utils_test.go

```diff
@@ -0,0 +1,68 @@
+package utils
+
+import (
+	"testing"
+)
+
+func TestGeneratePassHash(t *testing.T) {
```

Comment:
this test is essentially testing the `bcrypt` package.
leave it to that package's authors to test their implementation and remove this test